crypto/sig/src/lib.rs

```diff
@@ -85,6 +57,154 @@ mod tests {
     use mc_util_from_random::FromRandom;
     use mc_util_test_helper::run_with_several_seeds;
 
+    mod compat_20210122 {
+        use crate::*;
+        use core::convert::TryFrom;
+
+        const EXPECTED_SUCCESS_SECKEY: [&[u8; 32]; 3] = [
+            include_bytes!("testdata/20210121/expected_success/0/seckey.bin"),
```

Comment:
How were all these test files generated?

Reply:
I hacked master to write the raw bytes of its `run_with_several_seeds` tests to `/tmp`, then added them here.

I experimented with the idea of writing them to `$OUT_DIR`, but that isn't actually provided unless the crate contains a build script, which I didn't want to add for this purpose.

Reply:
Might be worth documenting in case this ever needs to be regenerated.

Reply:
OK